TESTING.md

```
@@ -0,0 +1,171 @@
+
+This project uses Playwright for end-to-end testing with a dedicated test database.
+
+## Quick Start
+
+1. **Run all e2e tests:**
+   ```bash
+   pnpm e2e:test:with-db
+   ```
+
+2. **Run tests without database setup:**
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)